quinn-h3/src/connection.rs

```diff
@@ -24,6 +24,7 @@ use crate::{
 };
 
 const RECV_ENCODER_INITIAL_CAPACITY: usize = 20480;
+const RECV_DECODER_INITIAL_CAPACITY: usize = 2048;
```

Comment:
Please move constants to the bottom of the module (can do it in an additional commit at the end).